### PR TITLE
Add draft DG planned enhancements

### PR DESCRIPTION
**Pull request by Weiennn**
I added the draft for DG planned enhancement. Feel free to update it with more pictures or details if needed.

### CODE REVIEW

File: docs/images/PlannedEnhancementSOError.png
Comment (Goh-Li-Ting) [inline docs/images/PlannedEnhancementSOError.png:1]: Would it be better for the images to just show the command box and output box? I think the current images are fine as well, just wondering if we could make it clearer which part of the image to look at.

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Review by ricketytoc — APPROVED** [review]
LGTM

File: docs/images/PlannedEnhancementSOError.png
Comment (Weiennn) [reply docs/images/PlannedEnhancementSOError.png:1]: Good idea, I've cropped it

**Pull request merged**